pinot-spi/src/main/java/org/apache/pinot/spi/ingestion/batch/BatchConfig.java

```diff
@@ -190,6 +193,10 @@ public boolean isAppendUUIDToSegmentName() {
     return _appendUUIDToSegmentName;
   }
 
+  public boolean isExcludeTimestampsFromSegmentName() {
```

Comment:
Make this consistent with config name
```suggestion
  public boolean isExcludeTimeInSegmentName() {
```